salt/kubernetes-common/openstack-config.jinja

```diff
@@ -1,7 +1,11 @@
 [Global]
 auth-url="{{ pillar['cloud']['openstack']['auth_url'] }}"
+{% if pillar['cloud']['openstack']['trust_id'] %}
+trust-id="{{ pillar['cloud']['openstack']['trust_id'] }}"
+{% else %}
 username="{{ pillar['cloud']['openstack']['username'] }}"
 password="{{ pillar['cloud']['openstack']['password'] }}"
+{% endif %}
```

Comment:
I don't think this is right, a trust still requires a username and password to be supplied, it's just that the user+pass used doesn't have to have direct access to the domain+project combination..